orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -88,7 +88,7 @@ std::vector<std::unique_ptr<GraphTransformer>> GeneratePreTrainingTransformers(
 
       transformers.emplace_back(onnxruntime::make_unique<GeluFusion>(compatible_eps));
       transformers.emplace_back(onnxruntime::make_unique<LayerNormFusion>(compatible_eps));
-      transformers.emplace_back(onnxruntime::make_unique<SimplifiedLayerNormFusion>(compatible_eps));
+      transformers.emplace_back(onnxruntime::make_unique<SimplifiedLayerNormFusion>(compatible_eps, config.allow_layer_norm_mod_precision));
```

Comment:
>allow_layer_norm_mod_precision [](http://example.com/codeflow?start=107&length=30)

do u need the same param in onnxruntime/core/optimizer/graph_transformer_utils.cc ?

Reply:
Since this is changing precision, I am not sure if we should enable it for inference. For now, we can add it to training and revisit enabling it for inference if need arises